docs/rfcs/001-updateable-bundled-packages.md

```diff
@@ -0,0 +1,100 @@
+
+### Rules for Updateable Bundled Packages
+
+Any package that opts into this behavior must follow one rule: **its `engines` field must be regularly updated to reflect the necessary Atom version for the Atom, Electron, and Node.js APIs used in the package**.  This field defines the range of Atom versions in which the package is expected to work.  The field should always be set to the lowest possible Atom version that the package supports.
+
+If a package wants to use API features of a newer version of Atom while still supporting older Atom versions, it must do so in a way that is aware of the user's version and adjust itself accordingly.
+
+## Drawbacks
+
+> Why should we *not* do this?
+
+The primary drawback of this approach is that updateable bundled packages might exhibit problems on older Atom versions due to missing or changed APIs in Atom, Electron, or Node.js.  The solution for these packages is to keep their `engines` field updated appropriately, but there's still a chance that some updates will slip through without the necessary engine version changes.
+
+One other possible drawback is that an updated version of a bundled package might not be compatible across two different Atom channels.  For example, if the user installs a new update to a bundled package that only supports the current Atom Beta release or higher, the user will no longer have access to that package if they open Atom Stable.
+
+However, this drawback is no different than what the user would face today installing a community package under the same circumstances, so this could be considered a general problem in the Atom package ecosystem.
+
+## Rationale and alternatives
+
+> - Why is this approach the best in the space of possible approaches?
+> - What other approaches have been considered and what is the rationale for not choosing them?
+> - What is the impact of not doing this?
+
+This is the best approach for updating bundled packages because it allows those packages to take control of their own release cycle so long as they manage their Atom engine version correctly.  It also does so in a way that allows us to decide which packages can be updated independently, reducing the likelihood of problems for users.
+
+The primary alternative to this approach is to speed up the Atom release cycle so that bundled Atom package updates will reach users more frequently.  This approach will be investigated independently of this RFC as it may still be valuable even with updateable bundled packages.
+
+## Unresolved questions
+
+> - What unresolved questions do you expect to resolve through the RFC process before this gets merged?
+
+Is it enough to just depend on the `engines` field of `package.json` to protect users from installing a package update that doesn't work with their version of Atom?
+
+Is `updateable` the right name for the field in `package.json`?  Is there a clearer name?
```

Comment:
Naming things is hard 😆 

I think `updateable` is fine, personally.

Reply:
Maybe `coreUpdateable`? We don't want community package authors looking at a core package as an example to see that and think "Oh, I need to add this or my package won't get updates!".

Reply:
We can update our documentation and maybe even write a blog post to communicate that `updateable` is only for atom core packages.

Reply:
![screen shot 2018-05-22 at 10 04 03 am](https://user-images.githubusercontent.com/2988/40367492-90b82412-5da7-11e8-9bd2-8e68a788b194.png)

😆 :trollface:

Reply:
I like `coreUpdateable`, makes it a bit more clear.

On the topic of `updateable` vs `updatable`, somehow without the 'e' it looks weird to me but I can be convinced of `updatable` if more folks prefer that.